src/pages/home/report/ReportActionItem.js

```diff
@@ -134,13 +144,15 @@ class ReportActionItem extends Component {
                                         isVisible
                                         reportID={-1}
                                         reportActionID={-1}
+                                        onMarkAsUnread={() => {}}
```

Comment:
It would be better to default this to an empty function by using `defaultProps` of `ReportActionContextMenu`  vs explicitly passing an empty function.